### PR TITLE
Sign-In Widget (3rd generation) GA

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds guide about customization 3rd gen of SIW
- **Is this PR related to a Monolith release?** Yes. 2023.10.0

### Resolves:

* [OKTA-561049](https://oktainc.atlassian.net/browse/OKTA-561049)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+* Color contrast
+* Focus management
+* Screen reader behavior
+
+See [Sign-In Widget, third generation](https://help.okta.com/okta_help.htm?type=oie&id=ext-compare-siw) to verify if the third generation is right for your org.
+
+## Upgrade to the third generation
+
+> **Note:** If you have a new Okta Identity Engine org, the third-generation Sign-In Widget isn't enabled by default.
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:49]:
```suggestion
> **Note:** If you have a new Okta Identity Engine org, the third generation Sign-In Widget isn't enabled by default.
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+## Use design tokens
+
+Design tokens make the Sign-In Widget's visual style consistent and easier to update. Tokens replace static values to customize the following:
+
+* color
+* font
+* line height
+* size
+* tracking
+* weight
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:89]: Not sure what "tracking" means. FYC:

* colors
* borders
* font family
* font size
* font weight
* line height
* spacing

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+The following is a list of available design
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md]:
```suggestion
The following is a list of available design tokens:
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+### Color change
+
+* Button `#4C64E1` to `#D11DCA`
+* Text `E272727` to `#00297A`
+
+**Before color changes**
+
+<div class="half">
+
+![Your Alt Text](/img/siw-gen3/siw3-color-before.png)
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:173]: Do you typically leave the "Your Alt Text" placeholder?

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ To update the widget:
 
 5. Verify the CSS and localization changes that you made are reflected in the new version.
 
-  > **Note:** The third generation of the Okta Sign-In Widget doesn’t guarantee the stability of CSS selectors. Instead, customization in the third generation gets better support through branding. See [Customizations](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/).
+  > **Note:** The third generation of the Okta Sign-In Widget doesn’t guarantee the stability of CSS selectors. Instead, customization in the third generation gets better support through design tokens. See [Style the Sign-In Widget (third generation)](/docs/guides/custom-widget-gen3/).
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:73]: We still support branding

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md (hunk 4 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md]: Good catch. Should be null, or have alt text. I've fixed it.

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+* Font family
+* Font size
+* Font weight
+* Line height
+* Spacing
+
+The following is a list of available design tokens with default values. See [Customization examples](#customization-examples):
+
+```json
+    {
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:95]:
```suggestion
{
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+<div class="half">
+
+![After color changes](/img/siw-gen3/siw3-color-after.png)
+
+</div>
+
+### Text change
+
+* Heading Inter: `semibold 22` to `Helvetica, bold, 25`
+* Text Inter: `semibold, 14` to `Helvetica, regular, 16`
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:189]:
```suggestion
* `TypographyFamilyHeading` from `"'Aeonik', 'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif"` (default) to `"Helvetica"`
* `TypographyWeightHeading` from `500` (semibold, default) to `600` (bold)
* `TypographyFamilyBody` from `"'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif"` (default) to  `"Helvetica"`
* `TypographySizeBase` from `"87.5%"` (14px, default) to `"100%"` (16px)
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 8]
```diff
@@ -0,0 +1,247 @@
...
+<div class="half">
+
+![After color and text changes](/img/siw-gen3/siw3-color-text-after.png)
+
+</div>
+
+### Border radius and spacing change
+
+* Input/button border radius: `4px` to `24px`
+* Vertical spacing between elements in the main content area: `24px` to `40px`
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:218]:
```suggestion
* `BorderRadiusMain` from `4px` to `24px`
* `Spacing5` from `1.71428571rem` (24px, default) to `2.85714286rem` (40px)
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 9]
```diff
@@ -0,0 +1,247 @@
...
+```
+
+## Customization examples
+
+The following examples illustrate the impact of basic changes:
+
+### Color change
+
+* Button `#4C64E1` to `#D11DCA`
+* Text `E272727` to `#00297A`
```
Comment (lesterchoi-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:168]:
```suggestion
* `TypographyColorBody` from `#272727` (dark grey) to `#00297A` (navy blue)
```

**Review by lesterchoi-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 10]
```diff
@@ -0,0 +1,247 @@
...
+
+**Before color changes**
+
+<div class="half">
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:189]: It sets image size.

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 11]
```diff
@@ -0,0 +1,266 @@
...
+The Sign-In Widget enables the following:
+
+* Registration
+* Enrollment
+* Verification
+* Recovery experiences for your users
+
+It's where everyone in your org starts their Okta session.
+
+The third generation of the Sign-In Widget offers the same user experience as the second generation. It adds accessibility improvements in the following:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:39]:
```suggestion
The third generation of the Sign-In Widget offers the same user experience as the second generation. It adds accessibility improvements to the following:
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 12]
```diff
@@ -0,0 +1,266 @@
...
+## Use the code editor
+
+Use the code editor to add design tokens to your code. Design tokens call a JavaScript API.
+
+> **Note:** You can only enable the code editor if you configure a [custom domain](/docs/guides/custom-url-domain/).
+
+See [Customization examples](#customization-examples) for snippets that you can update and use.
+
+1. In the Admin Console, go to **Customizations** > **Brands**, and then select the brand you want.
+2. In the **Pages** tab, click **Configure** in the **Sign-in page** panel.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:72]:
```suggestion
2. On the **Pages** tab, click **Configure** in the **Sign-in page** panel.
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 13]
```diff
@@ -0,0 +1,266 @@
...
+3. To open the code editor, turn on the toggle next to **Code editor**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:73]:
```suggestion
3. To open the code editor, click the **Code editor** toggle.
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md [hunk 14]
```diff
@@ -0,0 +1,266 @@
...
+4. Make changes directly in the editor. See [Use design tokens](#use-design-tokens).
+   * Click **Save to draft**, then **Preview**, **Revert**, or **Publish**.
+   * Select **Compare with published version** to see the difference between your edited version and the published version. You can choose between a split view and a unified view.
+
+   > **Note:** To discard your changes without publishing them, click **Revert changes** or turn off the toggle next to **Code editor**. Turning off the code editor restores the default JavaScript code.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md:78]:
```suggestion
   > **Note:** To discard your changes without publishing them, click **Revert changes** or click the **Code editor** toggle again. Turning off the code editor restores the default JavaScript code.
```

**Review by susanharper-okta — APPROVED** [review]
just a couple baby nits....approving to not hold you up

**Pull request merged**